library/ubuntu

```diff
@@ -2,7 +2,7 @@
 # see also https://wiki.ubuntu.com/Releases#Current
 
 Maintainers: Tomáš Virtus <tomas.virtus@canonical.com> (@woky), Cristóvão Cordeiro <cristovao.cordeiro@canonical.com> (@cjdcordeiro)
-GitRepo: https://git.launchpad.net/cloud-images/+oci/ubuntu-base
+GitRepo: https://git.launchpad.net/cloud-images/+oci/ubuntu-base.git
```

Comment:
This one is actually the example URL I gave to the Moby folks internally -- Launchpad canonically does not use `.git` and does not support it (hence the CI failures on this PR :smile:)